recipes/python-asana/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-asana" %}
```

Comment:
Note: the PyPI name for this package is just `asana` but the GitHub repo is `python-asana` so I am deferring to the developers on this (with the assumption that there could be Asana packages for other languages).

Reply:
Agreed. Can you make a PR to Grayskull later so that it's able to find it correctly?

See this one as an example: https://github.com/conda-incubator/grayskull/pull/311/files

Reply:
Oh, neat! I didn't know about that ability in grayskull.